SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/SpatialGDKEditorSchemaGenerator.cpp

```diff
@@ -594,6 +594,14 @@ SPATIALGDKEDITOR_API bool GeneratedSchemaFolderExists()
 	return PlatformFile.DirectoryExists(*SchemaOutputPath);
 }
 
+SPATIALGDKEDITOR_API bool GeneratedSchemaDatabaseExists()
+{
+	const FString SchemaDatabasePackagePath = TEXT("/Game/Spatial/SchemaDatabase");
```

Comment:
This path is not consistent across projects, 'Game' is just what it is in the GDK projects.
I believe you can use `FPaths::ProjectContentDir()`

Reply:
Cheers, I'll amend that.

Reply:
I believe this should be resolved now.